tests/components/sensor/test_darksky.py

```diff
@@ -44,10 +91,41 @@ def tearDown(self):  # pylint: disable=invalid-name
         """Stop everything that was started."""
         self.hass.stop()
 
-    def test_setup_with_config(self):
+    @MockDependency('forecastio')
+    @patch('forecastio.load_forecast', new=load_forecastMock)
+    def test_setup_with_config(self, mock_forecastio):
         """Test the platform setup with configuration."""
-        self.assertTrue(
-            setup_component(self.hass, 'sensor', {'darksky': self.config}))
+        assert setup_component(self.hass, 'sensor', VALID_CONFIG_MINIMAL)
+
+        state = self.hass.states.get('sensor.dark_sky_summary')
+        assert state is not None
+
+    @MockDependency('forecastio')
+    @patch('forecastio.load_forecast', new=load_forecastMock)
+    def test_setup_with_invalid_config(self, mock_forecastio):
+        """Test the platform setup with invalid configuration."""
+        assert setup_component(self.hass, 'sensor', INVALID_CONFIG_MINIMAL)
```

Comment:
Since the assertion doesn't work in this case, maybe just remove the `assert`?

Reply:
Good idea.